### DISCUSSION TITLE
Unable to run Sample main.cpp According to the Latest 3.5 Documentation

### DISCUSSION THREAD

**Discussion by ZzWDlyy** [pelicanmapping/osgearth#2399] [Support Q&A] [answered]
Hello! I am a beginner in osgEarth. I followed the latest osgEarth 3.5 documentation and installed it using vcpkg. However, when I run the sample main.cpp, it displays the following:
![E%2ISWSS4KC3A M4LVT3$J](https://github.com/gwaldron/osgearth/assets/112188036/38c5b54f-eba9-4400-9ab8-144f6e7b0fbe)
Is anything wrong?Looking forward to your reply.

**Comment by gwaldron** [comment, marked as answer]
This bug was fixed prior too the 3.5 release. Please update your repo and try again :)
